### PR TITLE
Feature add params

### PR DESCRIPTION
**Pull request by vdejager**
I've added some code to include the 'host' and 'mode' parameters for the dash app launch. I needed this to get jbrowse-jupyter to play nice with a hosted jupyterlab environment behind a nginx/jupyterhub instance. 
I've also updated the imports in the example in README.md to reflect the recommendations given by dash as I tried to run them in a notebook

### CODE REVIEW

**Comment by teresam856** [comment]
@vdejager thank you for the contributions. Will go over the PR shortly and review it.

File: setup.py [hunk 1]
```diff
@@ -9,7 +9,7 @@
 
 setuptools.setup(
     name="jbrowse-jupyter",
-    version="1.2.1",
+    version="1.2.1.1",
```
Comment (teresam856) [inline setup.py:12]: We can make this version="1.2.3". Version 1.2.2 is out now.

File: jbrowse_jupyter/util.py [hunk 2]
```diff
@@ -194,5 +201,5 @@ def launch(conf, **kwargs):
                 )])
     else:
         raise TypeError(f'The {dash_comp} component is not supported.')
-    app.run_server(port=comp_port,
-                   height=comp_height, mode="inline", use_reloader=False)
+    app.run_server(port=comp_port, host=comp_host,
```
Comment (teresam856) [inline jbrowse_jupyter/util.py:204]: 👍

**Comment by vdejager** [comment]
Should be good to merge now.

**Comment by teresam856** [comment]
Will try to push another release in the coming days.

**Pull request merged**